src/utils/generate-site.js

```diff
@@ -150,8 +150,10 @@ async function createHTMLFile(templateObj, programInfo, options) {
     });
   }
 
-  // Execute beforeHTMLWrite plugins
-  htmlOut = await executeBeforeHTMLWritePlugins(htmlOut, programInfo);
+  // Execute beforeHTMLWrite plugins if the --ignore-plugins flag is not passed
+  if (!programInfo.ignorePlugins) { 
```

Comment:
Seems like an end of line issue with my file?
I'll be glad to help after someone reviews it.

Reply:
I'll just check locally and let you know

Reply:
I pushed a change. It should fix it. In future if you come across an eslint issue, you can fix it with `npx eslint . --fix`.